src/aui-form-builder/assets/aui-form-builder-field-list-core.css

```diff
@@ -2,50 +2,88 @@
     background-color: #ecf6fe;
     border: 1px solid #65b3f0;
     border-radius: 2px;
-    display: none;
-    margin: 0 auto;
-    opacity: 0.5;
+    visibility: hidden;
     text-align: center;
-    width: 92%;
+    cursor: pointer;
 }
```

Comment:
This code above makes the add button gets the same width of the fields.
I need use visibility property instead display property because its position is absolute now.